grails-app/migrations/views/cycle-count-session.sql

```diff
@@ -8,7 +8,7 @@ SELECT
     product_availability.location_id                                                 as facility_id,
 
     -- Cycle count request ID
-    cycle_count_request_summary.cycle_count_request_id                               as cycle_count_request_id,
+    cycle_count_request_summary.cycle_count_request_id                              as cycle_count_request_id,
```

Comment:
was this intended?

Reply:
nope, I was adding a workaround to make the view working after Justin's changes and after removing it, one of the indents stayed there. I removed it.